extension/src/test/suite/experiments/index.test.ts

```diff
@@ -700,6 +700,62 @@ suite('Experiments Test Suite', () => {
 
       expect(messageSpy).to.be.calledWith(allColumnsUnselected)
     })
+
+    it('should be able to handle a message to focus the sorts tree', async () => {
+      const { experiments } = buildExperiments(disposable, expShowFixture)
+
+      const webview = await experiments.showWebview()
+
+      const mockSendTelemetryEvent = stub(Telemetry, 'sendTelemetryEvent')
+      const mockMessageReceived = getMessageReceivedEmitter(webview)
+
+      mockMessageReceived.fire({
+        type: MessageFromWebviewType.FOCUS_SORTS_TREE
+      })
+
+      const executeCommandStub = stub(commands, 'executeCommand')
+
+      expect(executeCommandStub).to.be.calledWith(
+        'dvc.views.experimentsSortByTree.focus'
+      )
+      expect(mockSendTelemetryEvent).to.be.calledOnce
+      expect(
+        mockSendTelemetryEvent,
+        'should send a telemetry call that the sorts tree has been focused'
+      ).to.be.calledWithExactly(
+        EventName.VIEWS_EXPERIMENTS_TABLE_FOCUS_SORTS_TREE,
+        undefined,
+        undefined
+      )
+    })
```

Comment:
[B] Need to wait for the event to be flushed. Dirty version of the required test is 
```
    it('should be able to handle a message to focus the sorts tree', async () => {
      const { experiments } = buildExperiments(disposable, expShowFixture)

      const webview = await experiments.showWebview()

      const mockSendTelemetryEvent = stub(Telemetry, 'sendTelemetryEvent')
      const mockMessageReceived = getMessageReceivedEmitter(webview)

      const messageReceived = new Promise(resolve =>
        disposable.track(mockMessageReceived.event(() => resolve(undefined)))
      )

      const executeCommandStub = stub(commands, 'executeCommand')
      mockMessageReceived.fire({
        type: MessageFromWebviewType.FOCUS_SORTS_TREE
      })

      await messageReceived

      expect(executeCommandStub).to.be.calledWith(
        'dvc.views.experimentsSortByTree.focus'
      )
      expect(mockSendTelemetryEvent).to.be.calledOnce
      expect(
        mockSendTelemetryEvent,
        'should send a telemetry call that the sorts tree has been focused'
      ).to.be.calledWithExactly(
        EventName.VIEWS_EXPERIMENTS_TABLE_FOCUS_SORTS_TREE,
        undefined,
        undefined
      )
    })

```